v2/setup/images-control.sh

```diff
@@ -12,7 +12,6 @@ if [ "$(etcdctl get images-control-bootstrapped)" == "true" ]; then
 fi
 etcdctl set images-control-bootstrapped true
 
-etcdctl set /images/chronos      "mesosphere/chronos:chronos-2.4.0-0.1.20150828104228.ubuntu1404-mesos-0.27.0-0.2.190.ubuntu1404"
 etcdctl set /images/fd           "behance/flight-director:latest"
```

Comment:
maybe just in case we can get the exact sha of flight-director just in case "latest" for some reason has moved where it currently is? I know if for some reason flight-director version increases, it'll break on portfolio.